app/schemas/user.py

```diff
@@ -6,6 +6,13 @@ class UserSchema(Schema):
     id = fields.Integer(dump_only=True)
     email = fields.Email()
     username = fields.String()
+    age = fields.Integer()
+    gender = fields.String()
+    age_of_onset = fields.Integer()
+    seizure_type = fields.String()
+    caregiver_name = fields.String()
+    caregiver_contact = fields.String()
+    institution = fields.String()
     password = fields.String(load_only=True)
 
```

Comment:
@Schade9 seems you forgot to add the profileImage field to the schema.. meaning even when its sent it won't be picked up and stored.

Reply:
@MutegekiHenry i have added it